cms/models.py

```diff
@@ -750,25 +750,28 @@ def products(self):
         return page_data
 
     def get_context(self, request, *args, **kwargs):
-        if request.user.is_authenticated:
-            user = request.user.email
-        else:
-            if "anonymous_session_id" not in request.session:
-                request.session["anonymous_session_id"] = str(uuid.uuid4())
-            user = request.session["anonymous_session_id"]
```

Comment:
So I was adding this UUID - but we're not using it. And I don't forsee an actual reason for it. What I did do in this PR is make it so ALL anonymous users will have the same tag - meaning we can very easily target not-logged-in users.